src/pyasm/search/sql_dumper.py

```diff
@@ -17,6 +17,7 @@
 import types
```

Comment:
I dunno why this is in yellow.. Just break it into 2 lines
import re
import sys